Melanzana.MachO/MachObjectFile.cs

```diff
@@ -26,15 +33,27 @@ public MachObjectFile(Stream stream)
 
         public IList<MachLoadCommand> LoadCommands { get; } = new List<MachLoadCommand>();
 
+        /// <summary>
+        /// For object files the relocation, symbol tables and other data are stored at the end of the
+        /// file but not covered by any segment/section. We maintain a list of these data to make it
+        /// easier to address.
+        /// 
+        /// For linked files this points to the real __LINKEDIT segment. We slice it into subsections
+        /// based on the known LinkEdit commands though.
+        /// </summary>
+        public IList<MachLinkEditData> LinkEditData { get; } = new List<MachLinkEditData>();
```

Comment:
Maybe we can add method to `MachLoadCommand` to enumerate all its linked data and make this lazy enumerator...